Weather/Services/Cities/CitiesService.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+
+
+struct City {
+    let cityId: String
+    let name: String
+    let region: String
+    let country: String
+    let latitude: Double
+    let longitude: Double
+}
+
+
+typealias CitiesServiceCompletion = (cities: [City]?, error: NSError?) -> Void
+
+
+protocol CitiesService {
+    func fetchCitiesWithText(text: String, completion: CitiesServiceCompletion)
```

Comment:
Just wondering, from a style point of view, should we do something more: 

-- func fetchCities(withName text: String, completion: CitiesServiceCompletion)

so caller would do:

service.fetchCity(withName: "santar", completion: completion)

just an idea after consulting:

https://github.com/raywenderlich/swift-style-guide